pkg/logql/engine.go

```diff
@@ -113,7 +113,7 @@ type Querier interface {
 type EngineOpts struct {
 	// TODO: remove this after next release.
 	// Timeout for queries execution
-	Timeout time.Duration `yaml:"timeout"`
+	Timeout time.Duration `yaml:"timeout" doc:"deprecated"`
```

Comment:
Possible enhancement for later: add a `deprecated` flag so we can easily identify these